cartesi-rollups/build-dapps/run-dapp.md

```diff
@@ -67,14 +67,32 @@ cd rollups-examples
 ```shell
 cd echo-python
 ```
-3. Build the Echo DApp:
+
+3. Check if your Docker supports the RISCV platform by running:
+
+```shell
+docker buildx ls
+```
+
+If you do not see `linux/riscv64` in the platforms list, install QEMU by runing::
```

Comment:
Hello @nicocartesi.
I noticed just a typo at the end of this phrase, `runing::` should be `running:`.
But the content is good to go, nice job man ;)